### PR TITLE
Simple investor test

### PR DESCRIPTION
**Pull request by vgalilei**
# Describe your changes 

At this PR I add an example of unit test 

# I follow best practices 

- [x] My code is covered by unit tests 
- [x] This PR is less than [300 lines of code](https://www.atlassian.com/blog/git/written-unwritten-guide-pull-requests)
- [x] My code [smells good](https://en.wikipedia.org/wiki/Code_smell) 
- [x] I have tested my PR locally and proven it works. 
- [x] There is [no copy-paste](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)

### CODE REVIEW

**Review by thinklikeadesigner — COMMENTED** [review]
Got it, looks good. will use to check code.

**Review by thinklikeadesigner — APPROVED** [review]
100%

File: backend/app.js [hunk 1]
```diff
@@ -31,5 +32,4 @@ mongoose.connect(
   },
 );
 
-app.listen(port);
-
+module.exports = app
```
Comment (McCambley) [inline backend/app.js:35]: LGTM

File: backend/server.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const app = require('./app');
```
Comment (McCambley) [inline backend/server.js:1]: Nice I was wondering about this part of the project

**Review by McCambley — APPROVED** [review]
This all looks goo! Nice footer implimentation.

**Pull request merged**